app/src/main/java/com/example/recyclerviewprogram/ItemAdapter.java

```diff
@@ -0,0 +1,47 @@
+package com.example.recyclerviewprogram;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+public final class ItemAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder>{
+    private Context context;
+
+    public ItemAdapter(Context context){
+        this.context = context;
+    }
+
+    @NonNull
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int index) {
+        return new RecyclerView.ViewHolder(
+                LayoutInflater.from(parent.getContext())
+                        .inflate(R.layout.items, parent, false)
+        ) {};
+    }
+
+    @SuppressLint("SetTextI18n")
+    @Override
+    public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int index) {
+        TextView name = holder.itemView.findViewById(R.id.name_items);
```

Comment:
Maybe we can add in this class data how ArrayList<User> users and init this in constructor and in this we can use this construction: 
User user = data.get(index)
holder.itemView.setText(user.getName());